tests/sentry/api/endpoints/test_sentry_app_avatar.py

```diff
@@ -13,19 +13,28 @@ def setUp(self):
         self.endpoint = "sentry-api-0-sentry-app-avatar"
         self.login_as(self.user)
 
+    def get_avatar(self, resp, is_color=True):
```

Comment:
These changes are because the `resp.data["avatars"]` list is unordered, so we can't assume the avatar with color=True is first.